roles/custom/matrix-bridge-hookshot/tasks/reset_encryptioon.yml

```
@@ -0,0 +1,12 @@
+---
+- name: Resetting Hookshot's crypto store
+  ansible.builtin.command:
+    cmd: |
+      {{ devture_systemd_docker_base_host_command_docker }} run
+      --rm
+      --name={{ matrix_hookshot_container_ident }}-reset-crypto
+      --user={{ matrix_user_uid }}:{{ matrix_user_gid }}
+      --cap-drop=ALL
+      -v {{ matrix_hookshot_base_path }}/config.yml:/config.yml
+      {{ matrix_hookshot_docker_image }} yarn start:resetcrypto
```

```suggestion
      --mount type=bind,src={{ matrix_hookshot_base_path }}/config.yml,dst=/config.yml
      {{ matrix_hookshot_docker_image }}
     yarn start:resetcrypto
```

`--mount` is less magical than `-v`, so it's safer to use. `-v` auto-creates source mount points (`{{ matrix_hookshot_base_path }}/config.yml`) as directories if they do not exist yet for some reason, which can be quite harmful. We should avoid using `-v`.

Also, we may add `,ro` to the `--mount` line, as I suppose the container will not need to modify the configuration.

---

thanks